impeller/entity/contents/content_context.h

```diff
@@ -293,15 +293,22 @@ struct ContentContextOptions {
     kIgnore,
     /// Overwrite the stencil content to the ref value. Used for resetting the
     /// stencil buffer after a stencil-then-cover operation.
-    kSetToRef,
-    /// Draw the stencil for the nonzero fill path rule.
     kNonZeroWrite,
```

Comment:
For clarity, I suggest the following convention. Make the `StencilMode` specify the phase in play clearly. Something like `k<Phase><Option>` perhaps. In my head, with stencil-then-cover, `kEvenOddWrite` would be `kStencilEvenOddFill`. You used the term "preparation" above. If you prefer that (I'm warming up to it too), `kPreparationEvenOddFill`. I found `kEvenOffWrite` too terse to follow initially. Ditto for the "cover" phase.

Reply:
You can then document the two phases in detail.

Reply:
Done.